momento-sdk/src/test/java/momento/sdk/auth/V2CredentialProviderTest.java

```diff
@@ -0,0 +1,97 @@
+package momento.sdk.auth;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+import momento.sdk.exceptions.InvalidArgumentException;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+class GlobalCredentialProviderTest {
+  private static final String ENV_VAR_NAME = "MOMENTO_TEST_GLOBAL_API_KEY";
+  private static final String TEST_V2_API_KEY =
+      "eyJhbGciOiJIUzUxMiIsInR5cCI6IkpXVCJ9.eyJ0IjoiZyJ9.LloWc3qLRkBm_djlOjXE8wNSENqOay17xHLJR5XIr0cwkyhhh8w_oBaiQDktBkOvh-wKLQGUKavSQuOwXEb2_g";
+  private static final String TEST_ENDPOINT = "test_endpoint";
+  private static final String ENDPOINT_ENV_VAR = "MOMENTO_TEST_ENDPOINT";
```

Comment:
same in this sdk, let's try to test with the default env vars if possible

```suggestion
  private static final String ENDPOINT_ENV_VAR = "MOMENTO_ENDPOINT";
```